src/lib/stores/marketplace.ts

```diff
@@ -0,0 +1,686 @@
+                entrypoint: 'lib/main.dart',
+                providerRootDirectory: 'dart/starter'
+            }
+        ],
+        instructions: `For documentation and instructions check out [README.md](https://github.com/appwrite/templates/tree/main/node/starter) file.`,
+        vcsProvider: 'github',
+        providerRepositoryId: 'templates',
+        providerOwner: 'appwrite',
+        providerBranch: 'main',
+        variables: [
+            {
+                name: 'APPWRITE_API_KEY',
+                description: `The API Key to talk to Appwrite backend APIs. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/getting-started-for-server">Learn More</a>`,
+                value: '',
+                placeholder: 'd1efb...aec35',
+                required: false
+            }
+        ]
+    },
+    {
+        icon: 'icon-open-ai',
+        id: 'prompt-chatgpt',
+        name: 'Prompt ChatGPT',
+        tagline: 'Ask question, and let OpenAI GPT-3.5-turbo answer.',
+        permissions: ['any'],
+        events: [],
+        cron: '',
+        timeout: 15,
+        runtimes: [
+            {
+                name: 'node-18.0',
+                commands: 'npm install',
+                entrypoint: 'src/main.js',
+                providerRootDirectory: 'node/prompt-chatgpt'
+            }
+        ],
+        instructions: `For documentation and instructions check out [README.md](https://github.com/appwrite/templates/tree/main/node/prompt-chatgpt) file.`,
+        vcsProvider: 'github',
+        providerRepositoryId: 'templates',
+        providerOwner: 'appwrite',
+        providerBranch: 'main',
+        variables: [
+            {
+                name: 'OPENAI_API_KEY',
+                description: `A unique key used to authenticate with the OpenAI API. Please note that this is a paid service and you will be charged for each request made to the API. <a class="u-bold" target="_blank" href="https://platform.openai.com/docs/quickstart/add-your-api-key">Learn More</a>`,
+                value: '',
+                placeholder: 'sk-wzG...vcy',
+                required: true
+            },
+            {
+                name: 'OPENAI_MAX_TOKENS',
+                description: `The maximum number of tokens that the OpenAI response should contain. Be aware that OpenAI models read and write a maximum number of tokens per API call, which varies depending on the model. For GPT-3.5-turbo, the limit is 4096 tokens. <a class="u-bold" target="_blank" href="https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them">Learn More</a>`,
+                value: '512',
+                placeholder: '512',
+                required: false
+            }
+        ]
+    },
+    {
+        icon: 'icon-perspective-api',
+        id: 'analyze-with-perspectiveapi',
+        name: 'Analyze with PerspectiveAPI',
+        tagline: 'Automate moderation by getting toxicity of messages.',
```

Comment:
```suggestion
        tagline: 'Automate moderation by getting toxicity of messages.',
```

Is this enough for a user to actually understand what this does? I'm a little confused.